src/python/pants/backend/python/rules/python_fmt.py

```diff
@@ -62,28 +72,58 @@ def run_black(
     Digest,
     DirectoriesToMerge(directories=tuple(all_input_digests)),
   )
+  yield BlackInput(config_path, resolved_requirements_pex, merged_input_files)
 
+
+def generate_black_pex_args(files: Set[str], config_path: str, *, check_only: bool) -> Tuple[str, ...]:
```

Comment:
Is this intended to be called outside of this module? If not, I might make it private.
```suggestion
def _generate_black_pex_args(files: Set[str], config_path: str, *, check_only: bool) -> Tuple[str, ...]:
```